astrobee/commands/freeFlyerPlanSchema.json

```diff
@@ -842,6 +842,25 @@
       ]
     },
 
+    {
+      "type": "CommandSpec",
+      "id": "GuestScience.restartGuestScience",
+      "isRapidNative": false,
+      "note": "This command is the equivalent to issuing the stop guest science command and then the start guest science command. Sometimes guest science APKs become unresponsive and this is a quick way to try to get it work again. <p/>See GuestScience.startGuestScience for more on the guest science life cycle.",
```

Comment:
I think "note" -> "notes"

Reply:
Good catch, thanks! Fixed and applied to both the astrobee and android prs.